recipes/click-loglevel/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "click-loglevel" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6ea082db60d0087d84fbed50e9c47c73282f7e50826285e5ae741ceaa7d7fbf1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - click >=6
+
+test:
+  requires:
+    - pytest
+    - pytest-cov
+  source_files:
+    - test
+    - tox.ini
+  imports:
+    - click_loglevel
+  commands:
+    - python -m pytest -vv test
+
+about:
+  home: https://github.com/jwodder/click-loglevel
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Update to `LICENSE` which is in the PyPI tarball.

Reply:
Did you mean to suggest an edit here?  Exactly what should `license_file` be changed to?

Reply:
```suggestion
  license_file: LICENSE
```